modules/local/annotate_uniprot.nf

```diff
@@ -6,14 +6,15 @@ process ANNOTATE_UNIPROT {
         '' }"
 
     publishDir(
-        path: "${params.outdir}/protein-annotations",
+        path: "${params.outdir}/protein_annotations",
```

Comment:
love to see it

Reply:
All part of what's coming next with the README/publishdir changes!